lib/agoneum/web/controllers/registration_controller.ex

```diff
@@ -6,14 +6,39 @@ defmodule Agoneum.Web.RegistrationController do
 
   action_fallback Agoneum.Web.FallbackController
 
+  def new(conn, _params) do
+    changeset = Account.change_user(%User{})
+    render(conn, "new.html", changeset: changeset)
+  end
+
   def create(conn, %{"user" => user_params}) do
-    with {:ok, %User{} = user} <- Account.create_user(user_params) do
-      {:ok, jwt, _full_claims} = Guardian.encode_and_sign(user, :token)
+    type = conn |> get_format() |> String.to_atom()
+    create(conn, user_params, type)
+  end
+  defp create(conn, user_params, :html) do
+    case Account.create_user(user_params) do
+      {:ok, %User{} = user} ->
+        conn
+        |> Guardian.Plug.sign_in(user)
+        |> redirect(to: page_path(conn, :index))
+      {:error, changeset} ->
+        conn
+        |> put_status(:unprocessable_entity)
+        |> render(:new, changeset: changeset)
+    end
+  end
+  defp create(conn, user_params, :json) do
```

Comment:
Just curious - do we want / need user creation through the api? I'm fine with leaving this web-based only - esp. since this will probably change up when introducing Oath.